app/api/issue_api.rb

```diff
@@ -42,6 +50,19 @@ def bbox_from_string(string, factory)
     end
     get '/issues' do
       scope = Issue.all.includes(:created_by, :tags)
+      if params[:group]
+        group = Group.where(short_name: params[:group]).first
```

Comment:
What is there is no group matching this name?  Currently this would 500 but we could add
`error!('400 Group not found', 400) unless group`